### ISSUE TITLE
Add support for ECC keys

### ISSUE THREAD

**Issue by oliver-koch** [lucanaef/PGPro#27]
Hello lucanaef,

i want to try your PGPro app to use GPG encryption on my iOS 14.1 device.
Unfortunatly i failed to import my GPG private and public key. 
Reason was "Unsupported Key".

I have generated my keys using the following command on a debian buster system:
```
gpg --quick-gen-key "Name <email>" ed25519 cert 2y
gpg --quick-add-key [keyid] cv25519 encr 1y
gpg --quick-add-key [keyid] ed25519 auth 1y
gpg --quick-add-key [keyid] ed25519 sign 1y
```

I use the following version of GnuPG 
```
gpg (GnuPG) 2.2.12
libgcrypt 1.8.4
```
Do you have an idea if  i make anything wrong, or is this "state of the art" key really not supportes? If second, will/can support for this kind of key be available in the near future?

Kind regards
Oliver Koch

**Comment by lucanaef** [comment]
Unfortunately ECC keys really are currently not supportet. I do want to support them in the future, but I can't tell you how long it's going to take.

**Comment by oliver-koch** [comment]
Thank you very much for your quick response. Would be greate to have the support of ECC keys inside you app.
Looking forward to the next version. :)

**Comment by nkbooth** [comment]
I have been using your application for some time. I have migrated my keys to ECC. I hope your support for these keys comes soon.

**Comment by therealpaulgg** [comment]
This app would be perfect for what I need if it had ECC support. It seems that ECC support was just recently added to the master branch of ObjectivePGP but not released yet. https://github.com/krzyzanowskim/ObjectivePGP/issues/190.

Seems that based on the code of the application it hopefully should just be a simple update of ObjectivePGP to get it working? Although I don't know when they'd actually release their newest version.

**Comment by oliver-koch** [comment]
Yes now it was possible. I will try it out. ✊🏻

**Comment by oliver-koch** [comment]
First Test Looks very Good! 😃
I will have a deeper Look tomorrow. 

![image](https://user-images.githubusercontent.com/64773331/137202937-bdd934f4-731a-4608-9c05-6bc50b9ec506.jpeg)

**Comment by lucanaef** [comment]
I've just released version 2021.4 which finally includes ECC support. If any problems occur, please (re-)open an issue.
